package/cluster/definition.yaml

```diff
@@ -3,6 +3,7 @@ kind: CompositeResourceDefinition
 metadata:
   name: xclusters.aws.platformref.upbound.io
 spec:
+  defaultCompositeDeletePolicy: Foreground
```

Comment:
Would this be sufficient without passing a special flag to kubectl?

Reply:
Yes, but only if we have a claim applied. It will not work if we create the XR without a claim.